visualization/app/codeCharta/ui/codeMap/codeMap.arrow.service.ts

```diff
@@ -103,7 +104,7 @@ export class CodeMapArrowService
 	}
 
 	private isEdgeApplicableForBuilding(codeMapBuilding: CodeMapBuilding) {
-		return this.storeService.getState().dynamicSettings.edgeMetric !== "None" && !codeMapBuilding.node.flat
+		return !this.storeService.getState().appSettings.edgeMetricToggler && !codeMapBuilding.node.flat
```

Comment:
Maybe the naming is a bit confusing beacuse I do not know if `edgeMetricToggler === false` means that the edge metric is disabled or vice verca. If name would be like `edgeMetricDisabled`, then it would be clear.